ci-operator/config/openshift/cluster-version-operator/openshift-cluster-version-operator-main.yaml

```diff
@@ -180,6 +184,23 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: openshift-e2e-azure
+- as: yamllint
+  commands: |
+    cfg="$(mktemp)"
+    cat >"$cfg" <<EOF
+    extends: default
+    rules:
+      document-start: disable
+      line-length: disable
+      indentation:
+        indent-sequences: consistent
+    EOF
+    yamllint -v
+    yamllint --strict -c "$cfg" ./install
```

Comment:
Can we do this in the repository `hack/verify-yaml.sh` script called a by a `make verify`, and just call that from here?

Reply:
https://github.com/openshift/cluster-version-operator/pull/1177